src/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
```

Comment:
- [ ] Kindly follow the duck pattern. And put your configureStore.js into the redux file.

Reply:
well resolved! ✔️

Reply:
well resolved! ✔️